package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/console-proxy",
-  "version": "0.0.7",
+  "version": "0.0.0-semantically-released",
```

Comment:
I think this change would be better off in 98d662c0d7c44c3feda9ae76a38aff610c74e565, since it is related to `semantic-release` and not really to `readme.md` update.